lang/cs/Org.Apache.REEF.Driver/DriverSubmissionSettings.cs

```diff
@@ -171,7 +171,7 @@ public bool RunOnYarn
             }
         }
 
-        public string ToComamndLineArguments()
```

Comment:
Public method in non-example non-test code => needs to go through "Obsolete in 0.14" path